File: Modul_1/C/PRAK110-2310817210013-Muhammad_Aufa_Fitrianda.c
#include <stdio.h>
#include <math.h>s
int main(){
int alas = 5;
int tinggi = 12;
int sisiMiring = (alas*alas)+(tinggi*tinggi);
int hasilSisiMiring = sqrt(sisiMiring);
int kelilingSegitiga = alas+tinggi+hasilSisiMiring;
int luasSegitiga = alas*tinggi/2;
printf("Diketahui :\n");
printf("Alas = %d cm\n",alas);
printf("Tinggi = %d cm\n\n", tinggi);
printf("Jawab :\n");
printf("Sisi A = %d cm\nSisi B = %d cm\nSisi C = %d cm\n",alas, hasilSisiMiring, tinggi);
printf("keliling = %d cm\n",kelilingSegitiga);
printf("luas = %d cm",luasSegitiga);
}

File: Modul_1/Python/PRAK110-2310817210013-Muhammad_Aufa_Fitrianda.py
import math
alas = 5
tinggi = 12
miring = round (math.sqrt(alas*alas+tinggi*tinggi))
keliling = round (alas+tinggi+miring)
luas = round (alas*tinggi/2)
print("Diketahui :")
print(f"Alas = {alas}cm")
print(f"Tinggi = {tinggi}cm")
print("Jawab :")
print(f"Sisi A = {alas} cm") 
print(f"Sisi B = {miring} cm")
print(f"Sisi C = {tinggi} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm")